hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1131,7 +1131,20 @@
   /** Conf key for enabling meta replication */
   public static final String USE_META_REPLICAS = "hbase.meta.replicas.use";
   public static final boolean DEFAULT_USE_META_REPLICAS = false;
+
+  /**
+   * @deprecated Since 2.4.0, will be removed in 4.0.0. Please change the meta replicas number by
+   *             altering meta table, i.e, set a new 'region replication' number and call
+   *             modifyTable.
+   */
+  @Deprecated
   public static final String META_REPLICAS_NUM = "hbase.meta.replica.count";
+  /**
+   * @deprecated Since 2.4.0, will be removed in 4.0.0. Please change the meta replicas number by
+   *             altering meta table, i.e, set a new 'region replication' number and call
+   *             modifyTable.
+   */
+  @Deprecated
```

Comment:
Doc needs an edit. It has section on meta replicas how-to.

Reply:
Grep hbase.meta.replica.count in the doc.

Reply:
Let me open a new issue to land the documentation change.

Reply:
https://issues.apache.org/jira/browse/HBASE-25169